oidc_provider/lib/utils/token.py

```diff
@@ -138,22 +134,3 @@ def create_code(user, client, scope, nonce, is_authentication,
     code.is_authentication = is_authentication
 
     return code
-
-
-def get_client_alg_keys(client):
```

Comment:
This function is no longer necessary as the way to get the key has been simplified.

Reply:
By removing this, and switching to the new 'simplified' method, explicit checking for specific algorithms is lost, which opens the door to future issues.  I would much prefer to see explicit checking added back - even if it adds some code complexity, the greater clarity in code paths and explicit exception for other algorithms is worth the cost.